libs/react-numerics/tsconfig.json

```diff
@@ -1,18 +1,27 @@
 {
   "extends": "../../tsconfig.base.json",
   "compilerOptions": {
-    "jsx": "react-jsx",
+    "jsx": "preserve",
+    "module": "esnext",
+    "moduleResolution": "node",
+    "target": "es5",
+    "lib": ["dom", "dom.iterable", "esnext"],
+    "noEmit": true,
+
+    "incremental": true,
+    "strict": false,
     "allowJs": true,
-    "esModuleInterop": true,
     "allowSyntheticDefaultImports": true,
+    "downlevelIteration": true,
+    "esModuleInterop": true,
     "forceConsistentCasingInFileNames": true,
-    "strict": true,
-    "noImplicitOverride": true,
-    "noPropertyAccessFromIndexSignature": true,
-    "noFallthroughCasesInSwitch": true
+    "isolatedModules": true,
+    "noFallthroughCasesInSwitch": true,
+    "resolveJsonModule": true,
+    "skipLibCheck": true
   },
   "files": [],
-  "include": [],
+  "include": ["@types", "src"],
```

Comment:
I'm not familiar with all these options but do they affect the ability build and publish?

These defaults are set by NX during setup. Does this break any of the NX scripts (I wouldn't expect them to)?

I don't understand, why is `strict` false?